caesars-palace/blocks/slider-card/slider-card.js

```diff
@@ -0,0 +1,140 @@
+export default function decorate(block) {
+  const cardWrapper = document.createElement('div');
+  cardWrapper.classList.add('card-wrapper');
+  block.querySelectorAll('div.slider-card > div').forEach((div) => {
+    div.classList.add('card');
+    const imageDiv = div.children[0];
+    imageDiv.children[0].children[3].classList.add('card-image');
+    div.children[1].classList.add('short-description');
+    div.children[2].classList.add('long-description');
+    const closeButton = document.createElement('div');
+    closeButton.classList.add('close-button');
+    closeButton.classList.add('hide');
+    div.insertBefore(closeButton, div.children[2]);
+  });
+
+  const shortDescriptionDivs = document.querySelectorAll('.short-description');
+  shortDescriptionDivs.forEach((div) => {
+    const title = div.children[0].children[0];
+    title.classList.add('title');
+    const showMore = div.children[1].children[0];
+    if (showMore) {
+      showMore.classList.add('show-more');
+    }
+  });
+
+  function isATablet() {
+    const mediaQuery = window.matchMedia(
```

Comment:
You can also use an event watcher, so it would switch properly between tablet/desktop mode while debugging. Here is an example of what I mean: https://github.com/hlxsites/caesars/blob/main/caesars-palace/blocks/highlight-card/highlight-card.js#L17-L40

Reply:
Do we need the event watcher here? 
I'm setting the scalingFactor based on whether or not it is a tablet.

Reply:
Imagine if you pivot your tablet screen. Some of the components need to switch to the "desktop" experience. If you do this check only once when decorating, it won't behave properly when switching between portrait/landscape. Since `decorate` is called only once when the page is created.

How I found this: When debugging, I change the browser window size with the mouse or the inspector, to verify it behaves correctly on resize.

Reply:
Done.